src/components/WeddingDetails/WeddingDetails.tsx

```diff
@@ -2,48 +2,47 @@ import './WeddingDetails.css';
 import { individualWedding } from '../../weddingData'
 import WeddingPhotoList from '../WeddingPhotoList/WeddingPhotoList';
 import {Link} from 'react-router-dom'
+import { StyledButton } from '../App/styledComponents.styles'
 
 type IndividualWedding = {
-		weddingId: number;
+	weddingId: number;
   name: string;
   image: string;
   date: string;
   email: string;
-  familyPhotoList: {name: string, photos: number[], phone: number}[] 
+  familyPhotoList: {name: string, photos: number[], phone: number}[]
   photoList: {photoId: number, guests: string[], description: string}[]
 }
 
 const WeddingDetails: React.FC<IndividualWedding> = ({
-	weddingId, 
+	weddingId,
 	name,
-	image, 
-	date, 
-	email, 
-	familyPhotoList, 
+	image,
+	date,
+	email,
+	familyPhotoList,
 	photoList
 }) => {
 
 	return (
 		<section className="detailsWrapper">
 			<div className="detailsHeader">
-				<h1>{name} Wedding</h1>
-				<h2>{date}</h2>
-				<p className="email">Email: {email}</p>
-				<p className="status">Status: {familyPhotoList.length === 0 ? "Pending" : "Received"}</p>
-			</div>
-			<section className="weddingImage">
-				<img src={image} />
-			</section>	
-			<div className="button" id="addWeddingButton">
-				<div id="translate"></div>
-				{photoList.length > 0 ? 
-					<Link to={`/${weddingId}/photo-list`}>View Photo List</Link> :
-					<Link to="/:weddingId/create-guest-list">Create Guest List</Link>
+				<h1 className="weddingTitle">{name} Wedding</h1>
+				<h2 className="weddingDate">{date}</h2>
+				<p className="weddingDetails">Email: {email}</p>
+				<p className="weddingDetails">Status: {familyPhotoList.length === 0 ? "Pending" : "Received"}</p>
+				<StyledButton>
```

Comment:
Pretty awesome that you were able to create and incorporate this style component. It will be helpful with future components

Reply:
Hoping we can utilize styled components a little more down the line. Maybe some styled inputs